lochness/rpms/__init__.py

```diff
@@ -31,12 +31,14 @@ def get_rpms_database(rpms_root_path) -> Dict[str, pd.DataFrame]:
                      value: pandas dataframe of the measure database
         
     '''
-    rpms_root_path = 'RPMS_repo'
-
     all_df_dict = {}
     for measure_file in Path(rpms_root_path).glob('*csv'):
         measure_name = measure_file.name.split('.')[0]
-        df_tmp = pd.read_csv(measure_file)
+        try:
+            df_tmp = pd.read_csv(measure_file)
+        except pd.errors.EmptyDataError:
```

Comment:
If the csv file is empty, skip it (this was the case in one of the examples given by Larry)